docs/running-on-kubernetes.md

```diff
@@ -132,6 +132,24 @@ To specify a main application resource that is in the Docker image, and if it ha
       --conf spark.kubernetes.executor.docker.image=registry-host:5000/spark-executor:latest \
       container:///home/applications/examples/example.jar
 
+### Setting Up SSL For Submitting the Driver
```

Comment:
There might be something we want to note here about certificates and hostname matching against the Kubelets.